backend/app/models/assistants.py

```
@@ -10,15 +10,19 @@
 class AssistantBase(SQLModel):
     assistant_id: str = Field(index=True, unique=True)
     name: str
-    instructions: str
+    instructions: str = Field(sa_column=Column(Text, nullable=False))
```

Ruff (0.12.2)

Undefined name `Text`

(F821)